flexmeasures/data/models/planning/tests/test_solver.py

```diff
@@ -1864,22 +1864,34 @@ def test_battery_storage_efficiency_sensor(
 @pytest.mark.parametrize(
     "sensor_name, expected_start, expected_end",
     [
+        # A value defined in a coarser resolution is upsampled to match the power sensor resolution.
         (
             "soc-targets (1h)",
             "14:00:00",
-            "14:45:00",
-        ),  # A value defined in a larger resolution is downsampled to match the power sensor resolution.
+            "15:00:00",
```

Comment:
Shouldn't this also include the period 13:45? I'm assuming the SOC of the scheduler represents the value at the end of the period, thus, a SOC constraint should be set such that the SOC at the instant right before the tick the SOC respects the constraint.

Reply:
Closing this as I was interpreting the SOC values to be defined at the end of the period instead of the start.